src/test/charts/functest/templates/ingress-EKS.yaml

```diff
@@ -16,7 +16,7 @@ metadata:
     "nginx.ingress.kubernetes.io/proxy-send-timeout": "60"
 spec:
   rules:
-    {{- range $serviceName := .Values.ingressNames }}
+    {{- range $serviceName := .Values.backdoorServiceNames }}
```

Comment:
Now that we're using the Ingress from the main chart when on EKS, we only need to create an ingress for the backdoor services in the functest chart